### PR TITLE
refactor: update links to the react subnets pages

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Update the links to subnets and subpages to use the React versions.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Load the UI and click on "Subnets" in the header. It should take you to the React networks page.
- Click on a subnet, vlan, space and fabric and you should get taken to the React pages.
- Click on a VLAN that has a rack controller.
- Click on the controller and you should get taken to the legacy controller page.
- Click on the VLANs tab.
- Click on a fabric, vlan and subnet and you should get returned to the React pages.

## Fixes

Fixes: canonical-web-and-design/app-tribe#677.

### CODE REVIEW

File: ui/src/app/base/urls.ts [hunk 1]
```diff
@@ -1,23 +1,5 @@
...
-  fabric: argPath<{ id: Fabric["id"] }>("/fabric/:id"),
-  images: "/images",
   index: "/",
-  intro: {
-    index: "/intro",
-    user: "/intro/user",
-  },
-  subnet: argPath<{ id: Subnet["id"] }>("/subnet/:id"),
-  vlan: argPath<{ id: VLAN["id"] }>("/vlan/:id"),
-  zone: argPath<{ id: Zone["id"] }>("/zone/:id"),
```
Comment (huwshimi) [inline ui/src/app/base/urls.ts:20]: There were some duplicate routes in this file that I removed.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM, but what is this `FETCH_HEAD` file?

**Comment by petermakowski** [comment]
> LGTM, but what is this `FETCH_HEAD` file?

@huwshimi I just noticed this file on `master`. Not sure what this is doing outside of `.git` directory, I'm assuming this was done by mistake? 

https://github.com/canonical-web-and-design/maas-ui/pull/3815

**Comment by huwshimi** [comment]
> > LGTM, but what is this `FETCH_HEAD` file?
> 
> @huwshimi I just noticed this file on `master`. Not sure what this is doing outside of `.git` directory, I'm assuming this was done by mistake?
> 
> #3815

Oops yeah I have no idea either (and somehow managed to miss @Caleb-Ellis's comment before I merged).

**Pull request merged**